### PR TITLE
wsgi: allow to run via socket

### PR DESCRIPTION
**Pull request by gmelikov**
Main part of code and modules already support it,
just check input args appropriately.

Signed-off-by: George Melikov <mail@gmelikov.ru>

### CODE REVIEW

**Comment by gmelikov** [comment]
This change allows us to create docker plugins on webpy.

File: web/httpserver.py [hunk 1]
```diff
@@ -168,10 +168,13 @@ def runsimple(func, server_address=("0.0.0.0", 8080)):
     
     server = WSGIServer(server_address, func)
 
-    if server.ssl_adapter:
-        print("https://%s:%d/" % server_address)
+    if '/' in server_address[0]:
```
Comment (anandology) [inline web/httpserver.py]: Wouldn't it be better to use `server_address.startswith('/')`?

Comment (gmelikov) [reply web/httpserver.py]: @anandology i've got this variant from https://github.com/webpy/webpy/blob/master/web/net.py#L156 , but i won't mind to change it, if you want.

Thank you for fast review!

**Comment by anandology** [comment]
Thanks @gmelikov.

**Pull request merged**